com.unity.netcode.adapter.utp/CHANGELOG.md

```diff
@@ -1,25 +1,25 @@
 # Changelog
+
 All notable changes to this package will be documented in this file. The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
-## [Unreleased]
+## [1.0.0-pre.4] - 2022-01-04
 
 ### Added
 
-- Added new 'Max Send Queue Size' configuration field in the inspector. This controls the size of the send queue that is used to accumulate small sends together and also acts as an overflow queue when there are too many in-flight packets or when other internal queues are full.
+- Added new 'Max Send Queue Size' configuration field in the inspector. This controls the size of the send queue that is used to accumulate small sends together and also acts as an overflow queue when there are too many in-flight packets or when other internal queues are full. (#1491)
```

Comment:
I'm sorry to bother you again @simon-lemay-unity :)
This line should stay, right?

Reply:
Yes, this one is correct.